props/saber_fett263_buttons.h

```diff
@@ -293,248 +660,3355 @@ class SaberFett263Buttons : public virtual PropBase {
+            PlayMenuSound("mpwrlock.wav");
+            break;
+          case 12:
+          // Maximum Clash Strength
+            PlayMenuSound("maxclash.wav");
+            break;
+        }
+        break;
+      case MENU_SWINGON:
+      case MENU_TWISTON:
+      case MENU_THRUSTON:
+      case MENU_STABON:
+      case MENU_FORCEPUSH:
+      case MENU_TWISTOFF:
+      case MENU_POWERLOCK:
+        if (direction > 0) {
+          choice_ = true;
+          PlayMenuSound("menable.wav");
+        } else {
+          choice_ = false;
+          PlayMenuSound("mdisable.wav");
+        }
+        break;         
+      case MENU_SWINGON_SPEED:
+        if (direction > 0) {
+          if (calc_ < 600) {
+            PlayMenuSound("mup.wav");
+            calc_ += 50;
+          }
+          if (calc_ >= 600) {
+            calc_ = 600;
+            PlayMenuSound("mmax.wav");
+          }
+        } else {
+          if (calc_ > 200) {
+            PlayMenuSound("mdown.wav");
+            calc_ -= 50;
+          }
+          if (calc_ <= 200) {
+            calc_ = 200;
+            PlayMenuSound("mmin.wav");
+          }
+        }
+        SayNumber(calc_, SAY_WHOLE);
+        break;
+      case MENU_FORCEPUSH_LENGTH:
+      case MENU_CLASH_DETECT:
+        if (direction > 0) {
+          if (calc_ < 10) {
+            PlayMenuSound("mup.wav");
+            calc_ += 1;
+          }
+          if (calc_ >= 10) {
+            calc_ = 10;
+            PlayMenuSound("mmax.wav");
+          }
+        } else {
+          if (calc_ > 1) {
+            PlayMenuSound("mdown.wav");
+            calc_ -= 1;
+          }
+          if (calc_ <= 1) {
+            calc_ = 1;
+            PlayMenuSound("mmin.wav");
+          }          
+        }
+        SayNumber(calc_, SAY_WHOLE);
+        break;
+      case MENU_MAX_CLASH:
+        if (direction > 0) {
+          if (calc_ < 16) {
+            PlayMenuSound("mup.wav");
+            calc_ += 1;
+          }
+          if (calc_ >= 16) {
+            calc_ = 16;
+            PlayMenuSound("mmax.wav");
+          }
+        } else {
+          if (calc_ > 8) {
+            PlayMenuSound("mdown.wav");
+            calc_ -= 1;
+          }
+          if (calc_ <= 8) {
+            calc_ = 8;
+            PlayMenuSound("mmin.wav");
+          }          
+        }
+        SayNumber(calc_, SAY_WHOLE);
+        break;         
+      case MENU_LOCKUP_DELAY:
+        if (direction > 0) {
+          if (calc_ < 1200) {
+            PlayMenuSound("mup.wav");
+            calc_ += 100;
+          }
+          if (calc_ >= 1200) {
+            calc_ = 1200;
+            PlayMenuSound("mmax.wav");
+          }
+        } else {
+          if (calc_ > 200) {
+            PlayMenuSound("mdown.wav");
+            calc_ -= 100;
+          }
+          if (calc_ <= 200) {
+            calc_ = 200;
+            PlayMenuSound("mmin.wav");
+          }
+        }
+        SayNumber(calc_, SAY_WHOLE);
```

Comment:
Might be nice if this said "milliseconds" after it?

Reply:
added, but only used for LOCKUP_DELAY as the other settings are not in millis()